style.css

```diff
@@ -103,10 +105,11 @@ header p {
 }
 
 .showcase {
-    margin: 0;
+    margin: 0 0 73.8px 0;
```

Comment:
pixels dont have decimals. round this to 74, as thats what the browser does anyway

Reply:
resolved in ee4eddb